manifests/cf-manifest/deployments/aws/000-cf-infrastructure.yml

```diff
@@ -241,14 +241,14 @@ jobs:
   - name: router_z1
     instances: 1
     networks:
-      - name: cf1
-        static_ips: (( static_ips(5, 6, 15, 16, 17, 18, 19, 20) ))
+      - name: router1
+        static_ips: (( static_ips(1) ))
```

Comment:
I would suggest that we allocate more static IPs for the routers, rather than only one.

The reason is that in the future, during a normal operation, we might want to increase the number of routers to cope with the load. That can be really easy if there are IPs already allocated (just change the # of instances), but if the IPs are not allocated, we need to re-render the manifest and apply it again.

I think it is fairly safe to set this to something like `static_ips(1, 2, 3, 4, 5, 6, 7, 8)`, as there are plenty of IPs and we have only the routers. Also, this way we keep the same setup than before.

I am open to consider that this should be out of this PR, but the change is small.